package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "uploadtest",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@storybook/addon-actions": "^3.4.11",
+    "@storybook/react": "^3.4.11",
+    "bootstrap": "^4.1.3",
+    "enzyme": "^3.7.0",
+    "eslint-config-airbnb": "^17.1.0",
```

Comment:
Unit testing and linting pacakges are preferred to be installed as `devDependencies`.

Reply:
Done!  ESlint gives an error stating storybook should be in dependencies and not devDependencies, but I agree it should be in devDependencies.